Bottom: 7f605411b60e6fe0e7413f48e89706e6a97b93fd
Top:    b4adc6bf28cb66b2b9bae2c4b12d56790dc1e184
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-22 22:40:41 +0100

Remove "=" shortcut to the djopt() thingy which keeps eating my design

ARR!! how much work have I lost to this key binding!









---

diff --git a/src/gpcb-menu.res b/src/gpcb-menu.res
index 8f6e893..ca691e0 100644
--- a/src/gpcb-menu.res
+++ b/src/gpcb-menu.res
@@ -331,13 +331,11 @@ MainMenu =
    {"Rip up all auto-routed tracks" RipUp(All)}
    -
    {"Optimize routed tracks"
-    {"Auto-Optimize" djopt(auto)  a={"Shift-=" "Shift<Key>="}}
     {"Debumpify" djopt(debumpify) }
     {"Unjaggy" djopt(unjaggy) }
     {"Vianudge" djopt(vianudge) }
     {"Viatrim" djopt(viatrim) }
     {"Ortho pull" djopt(orthopull) }
-    {"Simple optimization" djopt(simple)  a={"=" "<Key>="}}
     {"Miter" djopt(miter) }
     {"Puller" a={"Y" "<Key>y"} Puller() }
     {"Global Puller"
